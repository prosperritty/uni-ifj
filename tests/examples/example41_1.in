10
10010
100
str
