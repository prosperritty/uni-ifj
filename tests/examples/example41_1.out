10
10010
100
